src/features/account/components/AccountInfo.tsx

```diff
@@ -0,0 +1,43 @@
+import { CircularProgress, Heading, VStack } from '@chakra-ui/react';
+import type { FunctionComponent } from 'react';
+import { useEnsName } from 'wagmi';
+
+import { AccountRoleTags, AccountRole } from '.';
+import { JazzIcon } from '../../../components/JazzIcon';
+import { trimAddress } from '../../../utils/string';
+
+export type AccountInfoProps = {
+  address: string;
+};
+
+export const AccountInfo: FunctionComponent<AccountInfoProps> = ({
+  address,
+}) => {
+  const { data, isLoading } = useEnsName({
+    address: `${address}` as unknown as `0x${string}`,
```

Comment:
Why the type cast?

Reply:
We have a `Hex` type in `@metamask/utils` which is the same as this fyi.

Reply:
noted

Reply:
updated